src/avfoundation.cs

```diff
@@ -4090,6 +4399,7 @@ interface AVMetadata {
 		NSString CommonKeySoftware { get; }
 
 		[Field ("AVMetadataFormatQuickTimeUserData")]
+		[Obsolete ("Use 'AVMetadataFormat' enum values")]
```

Comment:
when using `[Obsolete]` put the API under `#if !XAMCORE_4_0`
If/when we release this new profile it should be exempt of obsolete API